src/AasxCsharpLibrary/AdminShellPackageEnv.cs

```diff
@@ -394,11 +394,26 @@ private static (AasCore.Aas3_0.Environment, Package) LoadPackageAasx(string fn,
                     }
 
                 if (originPart == null)
-                    throw (new Exception("Unable to find AASX origin. Aborting!"));
+                    xs = package.GetRelationshipsByType(
+                        "http://www.admin-shell.io/aasx/relationships/aasx-origin");
+                    foreach (var x in xs)
+                        if (x.SourceUri.ToString() == "/")
```

Comment:
I strongly recommend to use the **Equals** comparison method for strings.
```suggestion
                        if (x.SourceUri.Equals("/"))
```


- `==` Operator: The `==` operator compares the references of the string objects. If two strings have the same sequence of characters, but they are stored in different memory locations.
- `Equals()` Method: The `Equals()` method, on the other hand, compares the actual contents of the strings. It checks whether the sequences of characters in both strings are identical.